src/sentry/static/sentry/app/views/settings/projectDebugFiles/debugFileRow.tsx

```diff
@@ -85,39 +85,43 @@ const DebugFileRow = ({debugFile, showDetails, downloadUrl, onDelete}: Props) =>
       </Column>
       <RightColumn>
         <ButtonBar gap={0.5}>
-          <Access access={['project:releases']}>
```

Comment:
Which endpoint is this go to? The ProjectReleasePermission has `project:releases` as allowed on all operations.

Reply:
These lines handle downloading of DIF
https://github.com/getsentry/sentry/blob/486dcd6f9ba93e51bf49317c998d626071b578f6/src/sentry/api/endpoints/debug_files.py#L105-L107

Reply:
This was written a couple of years back and the reasoning behind it is that you might not want to allow regular members to access this endpoint as it exposes a sensitive source code.